pkg/cloudprovider/controller/nodecontroller_test.go

```diff
@@ -755,72 +857,83 @@ func TestSyncNodeStatusTransitionTime(t *testing.T) {
 				Status: probe.Failure,
 				Err:    nil,
 			},
-			expectedRequestCount:         2, // List+Update
-			expectedTransitionTimeChange: true,
+			expectedRequestCount:   2, // List+Update
+			expectedTransitionTime: fakeNow,
```

Comment:
Add a comment why this is fakeNow instead of node.CreationTimestamp. I guess this is the first transition, from Unknown (when node was just created) to Full?